locales/en/moves.json

```diff
@@ -19,7 +19,8 @@
         "heading": "Supporting documents",
         "heading_plural": "Supporting documents ({{count}})",
         "empty": "No documents uploaded"
-      }
+      },
+      "update_link": "Change <span class=\"govuk-visually-hidden\">{{ category }}</span>"
```

Comment:
You could use [i18next contexts](https://www.i18next.com/translation-function/context) to not end up with an empty span if no category is passed:

```
"update_link": "Change",
"update_link_with_description": "Change <span class=\"govuk-visually-hidden\">{{ description }}</span>"
```

and in the view:

```nunjucks
{{ t("update_link", {
  context: "with_description" if category,
  description: category
}) }}
```